tests_integration/test_procedure_coverage.py

```diff
@@ -26,40 +26,46 @@ def test_procedure_coverage_flow(_test_steps):
     _test_steps.snowpark_list_should_return_no_data()
 
     _test_steps.snowpark_init_should_initialize_files_with_default_content()
+    _test_steps.add_parameters_to_procedure("a: int, b: str")
     _test_steps.add_requirements_to_requirements_txt(["coverage"])
     _test_steps.requirements_file_should_contain_coverage()
     _test_steps.snowpark_package_should_zip_files()
 
+    parameters = "(a int, b string)"
     procedure_name = (
-        _test_steps.snowpark_create_with_coverage_wrapper_should_finish_succesfully()
+        _test_steps.snowpark_create_with_coverage_wrapper_should_finish_successfully(
```

Comment:
Do we know what will happen when an update will be with parameters `"(c int, d string)"`? Do we have any test case for such scenario?

Reply:
I totally forgot about it

Reply:
We agreed to create a separate task for fixing this case.